recipes/jinja2cpp/all/patches/fix-cmake-1.3.1.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ac3bf55..b6553db 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -305,7 +305,7 @@ install(
+         InstallTargets
+     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/static
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+     FILE_SET HEADERS
+ )
+ 
+diff --git a/src/helpers.h b/src/helpers.h
+index 3af280f..b1c31fe 100644
+--- a/src/helpers.h
++++ b/src/helpers.h
+@@ -32,12 +32,21 @@ struct MultiStringLiteral
+ #endif
+     }
+ 
++#if defined(_MSC_VER) && (_MSC_VER <= 1900)
++    template<typename CharT>
++    auto GetValueStr() const
++    {
++        auto memPtr = SelectMemberPtr<CharT, &MultiStringLiteral::charValue, &MultiStringLiteral::wcharValue>::GetPtr();
++        return std::basic_string<CharT>(this->*memPtr);
++    }
++#else
+     template<typename CharT>
+     constexpr auto GetValueStr() const
+     {
+         constexpr auto memPtr = SelectMemberPtr<CharT, &MultiStringLiteral::charValue, &MultiStringLiteral::wcharValue>::GetPtr();
+         return std::basic_string<CharT>(this->*memPtr);
+     }
++#endif
+ 
+     template<typename CharT, const char* MultiStringLiteral::*, const wchar_t* MultiStringLiteral::*>
+     struct SelectMemberPtr;
+diff --git a/thirdparty/CMakeLists.txt b/thirdparty/CMakeLists.txt
+index fc8f51d..49540d3 100644
+--- a/thirdparty/CMakeLists.txt
++++ b/thirdparty/CMakeLists.txt
+@@ -70,8 +70,8 @@ endif()
+ 
+ if (NOT DEFINED JINJA2_PRIVATE_LIBS_INT)
+     set(JINJA2CPP_PRIVATE_LIBS ${JINJA2CPP_PRIVATE_LIBS}
+-        Boost::variant Boost::filesystem Boost::algorithm Boost::lexical_cast Boost::json
```

Comment:
@toge , would you mind telling me why we removing boost::regex, boost::json?

boost::regex seems to significantly speed ups template loading?

Reply:
@rmorozov 
Oh, it's my mistake.
Thank you  for your review!

Reply:
I have added special option to choose which regex to use - default one is boost regex, but if it is not suitable feel free to override and use std regex as it was earlier

https://github.com/jinja2cpp/Jinja2Cpp/blob/4393211baa8ba69437d070be3c9fcba59aad6793/CMakeLists.txt#L15